### DISCUSSION TITLE
Render bundles flickering

### DISCUSSION THREAD

**Discussion by sseanwang** [gpuweb/gpuweb#3334] [Q&A] [answered]
When I play the example: https://austin-eng.com/webgpu-samples/samples/animometer

I found if I use the max number of numTriangles and enable the renderBundles, the screen start flickering a lot. 

I have the same issue when I trying to use renderBundles to accelerate the drawing, anyone know why it happens? Is there any solution except just reduce the number of things to be bundled?

**Comment by kainino0x** [comment, marked as answer]
This sounds like a browser bug. Please file a bug on the relevant browser:
https://github.com/gpuweb/gpuweb/blob/main/.github/ISSUE_TEMPLATE/browser.md

In your report, include `chrome://gpu` for Chromium browsers, or `about:support` for Firefox browsers, so we can see your browser version, OS, OS version, GPU, GPU driver version, etc.
